pkg/psmdb/mongo/mongo.go

```diff
@@ -71,7 +71,7 @@ func ReadConfig(ctx context.Context, client *mongo.Client) (RSConfig, error) {
 		return RSConfig{}, errors.Wrap(res.Err(), "replSetGetConfig")
 	}
 	if err := res.Decode(&resp); err != nil {
-		return RSConfig{}, errors.Wrap(err, "failed to decode to replSetGetConfig")
+		return RSConfig{}, errors.Wrap(err, "failed to decoge to replSetGetConfig")
```

Comment:
decode is the right one =)

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/719/commits/de7cb57eba854c2446adfdc36261fbb71ebc1b04